piglet/src/test/java/org/apache/calcite/test/PigRelTestBase.java

```diff
@@ -33,6 +34,11 @@ public void testSetup() throws Exception {
     final FrameworkConfig config = PigRelBuilderTest.config().build();
     converter = PigConverter.create(config);
   }
+
+  static void skipTestsOnWindows() {
+    Assume.assumeFalse("Skip: Pig/Hadoop tests do not work on Windows",
```

Comment:
(Alt) Can we move the skip condition to the `testSetup` method just above.

Reply:
Yes, this seems the cleanest solution to me. I'll go for it